ssl/ssl_quic.c

```diff
@@ -235,7 +235,7 @@ int quic_set_encryption_secrets(SSL *ssl, OSSL_ENCRYPTION_LEVEL level)
 {
     uint8_t *c2s_secret = NULL;
     uint8_t *s2c_secret = NULL;
-    size_t len;
+    int len;
```

Comment:
Prefer to keep this as `size_t`. If it needs to be negative, use `ssize_t`. If it needs to be passed to an `int*`, add another variable. Otherwise, cast down to `int` rather than up to `size_t`.

Reply:
The reason to use `int` is there is a bug in `if ((len = EVP_MD_size(md)) <= 0) {`, where if EVP_MD_size returns negative a number, it is larger than 0 because len is unsigned.
Lets change int to ssize_t.

Reply:
1983db1c42f64ef43872760f6e2dbd412901350a